### PR TITLE
i2720 migrate pas

### PR DESCRIPTION
**Pull request by kayiwa**
- add mysql variables
- add pas playbook

### CODE REVIEW

**Comment by acozine** [comment]
Related to #2737.

File: group_vars/pas/production.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+server:
```
Comment (acozine) [inline group_vars/pas/production.yml:2]: While you're doing this, do you want to create a `group_vars/pas/common.yml` file for shared settings like hte timezone?

File: playbooks/pas.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  hosts: pas_{{ runtime_env | default('staging') }}
+  remote_user: pulsys
+  become: true
+  vars_files:
+    - ../group_vars/pas/{{ runtime_env | default('staging') }}.yml
+    - ../group_vars/pas/vault.yml
+  vars:
+    - pas_upload_path: 'pas/pas-production'
+  roles:
+    - role: roles/datadog
```
Comment (acozine) [inline playbooks/pas.yml:14]:
```suggestion
    - role: roles/datadog
      when: runtime_env | default('staging') == "production"
```

**Review by acozine — COMMENTED** [review]
Same thing here about datadog. I also started thinking about deduplicating vars - do we want to institute the "quad file" vars, where sensitive vars go in `vault.yml`, shared visible vars go in `common.yml`, and env-specific vars go in `production.yml` and `staging.yml`?

**Comment by acozine** [comment]
CI failure is `Failure downloading https://nodejs.org/download/release//v16.14.0/node-v16.14.0-linux-x64.tar.gz, The read operation timed out` - there's nothing we can do except wait. We could consider adding Node to our download mirror.

**Review by carolyncole — APPROVED** [review]
Tested on production

**Pull request merged**